storage/tianmu/core/pack_str.cpp

```diff
@@ -755,7 +755,7 @@ void PackStr::LoadUncompressed(system::Stream *f) {
   ASSERT(data_.sum_len == sz, "bad pack! " + std::to_string(data_.sum_len) + "/" + std::to_string(sz));
 }
 
-bool PackStr::Lookup(const types::BString &pattern, uint16_t &id) {
```

Comment:
Here, Lookup is an lookup operation, not a type or flag. Therefore, `Lookup` is a proper name to identify what we will do in this function.

Reply:
Yeah, I revert it back to `Lookup`.